tests/spread/integration/ca-certificates/task.yaml

```diff
@@ -0,0 +1,16 @@
+summary: Integration tests for CA Certificates
+
+execute: |
+  # Test 1: Test that the ca-certificates.crt file is created and linked to /etc/ssl/certs/ca-certificates.crt
+  rootfs="$(install-slices ca-certificates_data python3.8_core)"
+  cp /etc/resolv.conf "${rootfs}/etc/"
+  chroot ${rootfs} /usr/bin/python3.8 -c "import urllib.request;urllib.request.urlopen('https://example.com')"
+
+  # Test 2: Test that the /etc/ca-certificates.conf file is created and update-ca-certificates works
+  rootfs=$(install-slices ca-certificates_data-with-certs ca-certificates_bins)
+  test -f ${rootfs}/etc/ca-certificates.conf
+  ln ${rootfs}/bin/bash ${rootfs}/bin/sh
+  mkdir -p ${rootfs}/dev
+  touch ${rootfs}/dev/null
+  chroot ${rootfs} /usr/sbin/update-ca-certificates
+  test $(find ${rootfs}/etc/ssl/certs -maxdepth 1 -type f ! -name 'ca-certificates.crt' | wc -l) -eq 0
```

Comment:
Is using `find` necessary here? (since it needs a new slice definition, but a new SDF is welcome anyway!)
Instead, you can use something like:
```
test $(ls -1 -p ${rootfs}/etc/ssl/certs | grep -v / | wc -l) -eq 1
```

Reply:
`find` is actually used within the `update-ca-certificates` script, hence it's needed:

```sh
d "$ETCCERTSDIR"
if [ "$fresh" = 1 ]; then
  echo "Clearing symlinks in $ETCCERTSDIR..."
  find . -type l -print | while read -r symlink
  do
    case $(readlink "$symlink") in
      $CERTSDIR*|$LOCALCERTSDIR*) rm -f "$symlink";;
    esac
  done
  find . -type l -print | while read -r symlink
  do
    test -f "$symlink" || rm -f "$symlink"
  done
  echo "done."
fi
```
(Ln 114-128)